packages/react-storage/src/components/StorageBrowser/Views/Controls/TableV2.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import { StorageBrowserElements } from '../../context/elements';
+import { CLASS_BASE } from '../constants';
+
+export const BLOCK_NAME = `${CLASS_BASE}__table`;
+export const TABLE_DATA_CLASS = `${BLOCK_NAME}__data`;
+export const TABLE_HEADER_CLASS = `${BLOCK_NAME}__header`;
+export const TABLE_HEADER_BUTTON_CLASS = `${BLOCK_NAME}__header__button`;
+export const TABLE_DATA_TEXT_CLASS = `${BLOCK_NAME}__data__text`;
+export const TABLE_DATA_BUTTON_CLASS = `${BLOCK_NAME}__data__button`;
+
+const { Span, Table, TableBody, TableHead, TableRow } = StorageBrowserElements;
+
+export interface TableProps<T, K> {
+  data?: TableData<T, K>;
+  renderColumnItem?: (props: K, index: number) => React.JSX.Element;
+  renderRowItem?: (props: T, index: number) => React.JSX.Element;
+}
+
+export interface TableData<T, K = {}> {
+  columns?: K[];
+  rows: T[][];
+}
+
+export const TableDataText = ({
+  children,
+}: {
+  children: React.ReactNode;
+}): JSX.Element => <Span className={TABLE_DATA_TEXT_CLASS}>{children}</Span>;
```

Comment:
I can remove this too. Was using this in another branch to test out migrating the locations view table